packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -700,3 +719,268 @@ test('extra give wantMintedInvitation', async t => {
+  ],
+  [
+    'published.psm.IST.DAI_axl.metrics',
+    '{"blockHeight":"9555443","values":["{\\"body\\":\\"{\\\\\\"anchorPoolBalance\\\\\\":{\\\\\\"brand\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"slot\\\\\\",\\\\\\"iface\\\\\\":\\\\\\"Alleged: DAI_axl brand\\\\\\",\\\\\\"index\\\\\\":0},\\\\\\"value\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"bigint\\\\\\",\\\\\\"digits\\\\\\":\\\\\\"100064908627531159731648\\\\\\"}},\\\\\\"feePoolBalance\\\\\\":{\\\\\\"brand\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"slot\\\\\\",\\\\\\"iface\\\\\\":\\\\\\"Alleged: IST brand\\\\\\",\\\\\\"index\\\\\\":1},\\\\\\"value\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"bigint\\\\\\",\\\\\\"digits\\\\\\":\\\\\\"0\\\\\\"}},\\\\\\"mintedPoolBalance\\\\\\":{\\\\\\"brand\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"slot\\\\\\",\\\\\\"index\\\\\\":1},\\\\\\"value\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"bigint\\\\\\",\\\\\\"digits\\\\\\":\\\\\\"100064908610\\\\\\"}},\\\\\\"totalAnchorProvided\\\\\\":{\\\\\\"brand\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"slot\\\\\\",\\\\\\"index\\\\\\":0},\\\\\\"value\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"bigint\\\\\\",\\\\\\"digits\\\\\\":\\\\\\"329326452472000000000000\\\\\\"}},\\\\\\"totalMintedProvided\\\\\\":{\\\\\\"brand\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"slot\\\\\\",\\\\\\"index\\\\\\":1},\\\\\\"value\\\\\\":{\\\\\\"@qclass\\\\\\":\\\\\\"bigint\\\\\\",\\\\\\"digits\\\\\\":\\\\\\"429391361082\\\\\\"}}}\\",\\"slots\\":[\\"board02656\\",\\"board02314\\"]}"]}',
+  ],
+];
+
+/**
+ * Sample of what appears in chain config file
+ *
+ * @type {AnchorOptions[]}
+ */
+const anchorAssets = [
+  {
+    keyword: 'USDC_axl',
+    proposedName: 'USD Coin',
+    decimalPlaces: 6,
+    denom: 'ibc/toyusdc',
+  },
+  {
+    keyword: 'USDT_axl',
+    proposedName: 'Tether USD',
+    decimalPlaces: 6,
+    denom:
+      'ibc/F2331645B9683116188EF36FC04A809C28BD36B54555E8705A37146D0182F045',
+  },
+  {
+    keyword: 'DAI_axl',
+    proposedName: 'DAI',
+    decimalPlaces: 18,
+    denom:
+      'ibc/3914BDEF46F429A26917E4D8D434620EC4817DC6B6E68FB327E190902F1E9242',
+  },
+];
+
+// XXX copied (with minor tweak) from packages/inter-protocol/test/test-gov-collateral.js
+const makeMockBankManager = t => {
+  /** @type {BankManager} */
+  const bankManager = Far('mock BankManager', {
+    getAssetSubscription: () => assert.fail('not impl'),
+    getModuleAccountAddress: () => assert.fail('not impl'),
+    getRewardDistributorDepositFacet: () =>
+      Far('depositFacet', {
+        receive: () => /** @type {any} */ (null),
+      }),
+    addAsset: async (denom, keyword, proposedName, kit) => {
+      t.log('addAsset', { denom, keyword, issuer: `${kit.issuer}` });
+      t.truthy(kit.mint);
+    },
+    getBankForAddress: () => assert.fail('not impl'),
+  });
+  return bankManager;
+};
+
+test('restore PSM: decode metrics, governance with old board IDs', async t => {
+  const toSlotReviver = makeHistoryReviver(chainStorageEntries);
```

Comment:
this is is testing `makeHistoryReviver`, not anything about PSM per se. (it's testing the shape of what was published to vstorage, but only assuming the fixture is correct.)

if you move makeHistoryReviver remember to move this test with it.